pennylane/transforms/batch_params.py

```diff
@@ -71,29 +71,29 @@ def circuit(x, weights):
 
     >>> batch_size = 3
     >>> x = np.linspace(0.1, 0.5, batch_size)
-    >>> weights = np.random.random((batch_size, 10, 3, 3))
+    >>> weights = np.random.random((batch_size, 10, 3, 3), requires_grad=True)
 
     If we evaluate the QNode with these inputs, we will get an output
     of shape ``(batch_size,)``:
 
     >>> circuit(x, weights)
-    [-0.30773348  0.23135516  0.13086565]
+    tensor([-0.30773348 0.23135516 0.13086565], requires_grad=True)

```

Comment:
```suggestion
    tensor([ 0.00800498,  0.2735391 , -0.24395442], requires_grad=True)
```

Making output deterministic.